test/testenv_docker_up.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Load integration test constants
+SCRIPT_DIR="$(dirname $(readlink -f ${BASH_SOURCE[0]}))"
+source "${SCRIPT_DIR}/testenv_constants.sh"
+
+# Function to verify that the graphql server is ready to take requests
+checkGqlServer() {
+
+  url="${1}"
+  max_time="${2}"
+
+  elapsedTime=0
+  until curl "$url" &>/dev/null
+  do
+
+    # Exit with error code 1
+    if [ $elapsedTime == $max_time ]; then
+
+      echo "${RED}${url}${NC} time limit reached"
+      return 1
+
+    fi
+
+    # Wait 2s and rety
```

Comment:
rety --> retry

Reply:
Done. I've made the message a bit more descriptive too.